promgen/tasks.py

```diff
@@ -12,6 +13,12 @@
 
 logger = logging.getLogger(__name__)
 
+@shared_task
+def index_alert(alert_pk):
+    alert = models.Alert.objects.get(pk=alert_pk)
+    labels = json.loads(alert.body)["commonLabels"]
```

Comment:
I think this could be written as `labels = alert.json.get('commonLabels')` since Alert model defines a `.json` property

Reply:
Fixed